scripts/system/create/edit.js

```diff
@@ -613,10 +613,18 @@ var toolBar = (function () {
             var entityIsLoadedCheck = function() {
                 isLoadedCheckCount++;
                 if (isLoadedCheckCount === MAX_LOADED_CHECKS || Entities.isLoaded(entityID)) {
-                    var naturalDimensions = Entities.getEntityProperties(entityID, "naturalDimensions").naturalDimensions
+                    var dimensionsToUse;
+                    if (POST_ADJUST_ENTITY_TYPES.indexOf(properties.type) !== -1) {
+                        // If it is a "Model", use natural dimensions...
+                        dimensionsToUse = Entities.getEntityProperties(entityID, "naturalDimensions").naturalDimensions;
+                    } else {
+                        // If it is not a "Model", use local dimensions...
+                        dimensionsToUse = Entities.getEntityProperties(entityID, "localDimensions").localDimensions;
```

Comment:
Do we even need this for non-model entities?  Can we just make it so this loaded check stuff is only called for models?